PerformanceTesting/ecl/02eb_smallsorts2.ecl

```diff
@@ -15,17 +14,121 @@
+//xversion algo='taskquicksort',numrows=96000,nothor
+//xversion algo='taskquicksort',numrows=128000,nothor
+//xversion algo='taskquicksort',numrows=192000,nothor
+//xversion algo='taskquicksort',numrows=256000,nothor
+//xversion algo='taskquicksort',numrows=384000,nothor
+//xversion algo='taskquicksort',numrows=512000,nothor
+
+//xversion algo='parquicksort',numrows=1000
+//xversion algo='parquicksort',numrows=2000
+//xversion algo='parquicksort',numrows=4000
+//xversion algo='parquicksort',numrows=6000
+//xversion algo='parquicksort',numrows=10000
+//xversion algo='parquicksort',numrows=12000
+//xversion algo='parquicksort',numrows=16000
+//xversion algo='parquicksort',numrows=24000
+//xversion algo='parquicksort',numrows=32000
+//xversion algo='parquicksort',numrows=48000,nothor
+//xversion algo='parquicksort',numrows=64000,nothor
+//xversion algo='parquicksort',numrows=96000,nothor
+//xversion algo='parquicksort',numrows=128000,nothor
+//xversion algo='parquicksort',numrows=192000,nothor
+//xversion algo='parquicksort',numrows=256000,nothor
+//xversion algo='parquicksort',numrows=384000,nothor
+//xversion algo='parquicksort',numrows=512000,nothor
+
+//xversion algo='mergesort',numrows=1000
+//xversion algo='mergesort',numrows=2000
+//xversion algo='mergesort',numrows=4000
+//xversion algo='mergesort',numrows=6000
+//xversion algo='mergesort',numrows=10000
+//xversion algo='mergesort',numrows=12000
+//xversion algo='mergesort',numrows=16000
+//xversion algo='mergesort',numrows=24000
+//xversion algo='mergesort',numrows=32000
+//xversion algo='mergesort',numrows=48000,nothor
+//xversion algo='mergesort',numrows=64000,nothor
+//xversion algo='mergesort',numrows=96000,nothor
+//xversion algo='mergesort',numrows=128000,nothor
+//xversion algo='mergesort',numrows=192000,nothor
+//xversion algo='mergesort',numrows=256000,nothor
+//xversion algo='mergesort',numrows=384000,nothor
+//xversion algo='mergesort',numrows=512000,nothor
+
+//xversion algo='heapsort',numrows=1000
+//xversion algo='heapsort',numrows=2000
+//xversion algo='heapsort',numrows=4000
+//xversion algo='heapsort',numrows=6000
+//xversion algo='heapsort',numrows=10000
+//xversion algo='heapsort',numrows=12000
+//xversion algo='heapsort',numrows=16000
+//xversion algo='heapsort',numrows=24000
+//xversion algo='heapsort',numrows=32000
+//xversion algo='heapsort',numrows=48000,nothor
+//xversion algo='heapsort',numrows=64000,nothor
+//xversion algo='heapsort',numrows=96000,nothor
+//xversion algo='heapsort',numrows=128000,nothor
+//xversion algo='heapsort',numrows=192000,nothor
+//xversion algo='heapsort',numrows=256000,nothor
+//xversion algo='heapsort',numrows=384000,nothor
+//xversion algo='heapsort',numrows=512000,nothor
+
 import ^ as root;
-algo := #IFDEFINED(root.algo, 'quicksort');
-numRows := #IFDEFINED(root.numrows, 100);
- 
+algo := #IFDEFINED(root.algo, 'parmergesort');
+numRows := #IFDEFINED(root.numrows, 512000);
+numIters := #IFDEFINED(root.numIters, 10000/5);
```

Comment:
I can't see that "root.numIters" appears at any version tag? Do we plan to add it or it is for manual testing purposes only?
(The format of that parameter's name "numIters" is a bit inconsistent with the similar  "numrows".)

Reply:
I can't remember to be honest.  These tests were written more than 6 months ago.
I will merge as is - it would allow it to be configured in a version if desired.